CONTRIBUTING.md

```diff
@@ -2,48 +2,74 @@
 
 Thank you for considering a contribution to Archivematica!
 This document outlines the code review process for Archivematica, along with our standards for new code contributions.
+Following these guidelines helps us assess your changes faster and makes it easier for us to merge you submission!
+
+There are many ways to contribute: writing tutorials or blog posts about your experience, improving the documentation, submitting bug reports, answering questions on the mailing list, or writing code which can be incorporated into Archivematica itself.
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**
 
-- [Code review](#code-review)
+- [Submitting bugs](#submitting-bugs)
+- [Submitting changes](#submitting-changes)
   - [Getting started](#getting-started)
   - [When to submit code for review?](#when-to-submit-code-for-review)
   - [Opening the pull request](#opening-the-pull-request)
   - [Discussion](#discussion)
   - [Cleaning up the commit history](#cleaning-up-the-commit-history)
-    - [Commits should be specific and atomic](#commits-should-be-specific-and-atomic)
-    - [Every commit should work](#every-commit-should-work)
-    - [Commit summaries should be concise](#commit-summaries-should-be-concise)
-    - [Commit messages should be as detailed as they need to be (and no more)](#commit-messages-should-be-as-detailed-as-they-need-to-be-and-no-more)
+  - [Small changes](#small-changes)
 - [Contribution standards](#contribution-standards)
   - [Style](#style)
     - [Some extra notes](#some-extra-notes)
     - [Exceptions](#exceptions)
     - [Working with legacy code](#working-with-legacy-code)
   - [Documentation](#documentation)
   - [Tests](#tests)
+  - [Commit History](#commit-history)
+    - [Commits should be specific and atomic](#commits-should-be-specific-and-atomic)
+    - [Every commit should work](#every-commit-should-work)
+    - [Commit summaries should be concise](#commit-summaries-should-be-concise)
+    - [Commit messages should be as detailed as they need to be (and no more)](#commit-messages-should-be-as-detailed-as-they-need-to-be-and-no-more)
 - [Contributor's Agreement](#contributors-agreement)
   - [Why do I have to sign a Contributor's Agreement?](#why-do-i-have-to-sign-a-contributors-agreement)
   - [How do I send in an agreement?](#how-do-i-send-in-an-agreement)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Code review
+## Submitting bugs
+
+If you find a security vulnerability, do NOT open an issue.
+Email ???@artefactual.com instead.
+
+Github Issues / Redmine is our preferred place to have bugs submitted.
+We also accept bugs submitted through the [mailing list](https://groups.google.com/forum/#!forum/archivematica-tech).
```

Comment:
I think we concluded that we would open up issue reporting on Github rather than Redmine, but a post to the mailing list is always welcome too (especially if unsure if it's a bug or some local problem)